### PR TITLE
planner: binding from history GA user doc

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Add document for the GA of SQL binding from history in v6.6.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12736
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-plan-management.md [hunk 1]
```diff
@@ -310,6 +307,16 @@ You can execute this statement to change the status of a binding. The default st
...
 
+#### Change binding status according to `sql_digest`
+
+In addition to changing the binding status according to a SQL statement, you can also change the binding status according to `sql_digest`:
+
+```sql
+SET BINDING [ENABLED | DISABLED] FOR SQL DIGEST 'sql_digest';
+```
+
+The binding status that can be changed by `sql_digest` and the effect are the same as those changed by [change binding status according to a SQL statement](#change-binding-status-according-to-a-sql-statement). If no binding is available for status changes, a warning message `can't find any binding for 'sql_digest'` is returned.
```
Comment (ran-huang) [inline sql-plan-management.md:318]:
```suggestion
The binding status that can be changed by `sql_digest` and the effect are the same as those changed [according to a SQL statement](#change-binding-status-according-to-a-sql-statement). If no binding is available for status changes, a warning message `can't find any binding for 'sql_digest'` is returned.
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

**Comment by TomShawn** [comment]
@fzzf678

File: sql-plan-management.md [hunk 2]
```diff
@@ -310,6 +307,16 @@ You can execute this statement to change the status of a binding. The default st
...
 
+#### Change binding status according to `sql_digest`
+
+In addition to changing the binding status according to a SQL statement, you can also change the binding status according to `sql_digest`:
+
+```sql
+SET BINDING [ENABLED | DISABLED] FOR SQL DIGEST 'sql_digest';
+```
+
+The binding status that can be changed by `sql_digest` and the effect are the same as those changed [according to a SQL statement](#change-binding-status-according-to-a-sql-statement). If no binding is available for status changes, a warning message `can't find any binding for 'sql_digest'` is returned.
```
[the remark this answers is not in the document]
Comment (fzzf678) [reply sql-plan-management.md:318]: Rest LGTM

**Review by fzzf678 — APPROVED** [review]

**Pull request merged**